js/skin.js

```diff
@@ -251,12 +251,12 @@ var Skin = createReactClass({
     }
     // Focusable elements on the player (this.domElement) are expected to have the
     // data-focus-id attribute, this is a convention used throughout this project.
-    var selector = '[' + CONSTANTS.KEYBD_FOCUS_ID_ATTR + ']:enabled';
-    var focusableElements = this.domElement.querySelectorAll(selector);
+    const selector = '[' + CONSTANTS.KEYBD_FOCUS_ID_ATTR + ']:enabled';
+    let focusableElements = this.domElement.querySelectorAll(selector);
 
     if (focusableElements.length) {
-      var firstFocusableElement = focusableElements[0];
-      var lastFocusableElement = focusableElements[focusableElements.length - 1];
+      let firstFocusableElement = focusableElements[0];
+      let lastFocusableElement = focusableElements[focusableElements.length - 1];
```

Comment:
Both elements provided by reference. It is not going to change down by code, so it's a const.

Reply:
\+